### PR TITLE
Entity blagiy

### PR DESCRIPTION
**Pull request by VadimBlagiy**

### CODE REVIEW

File: sandbox-backend/src/main/java/com/exadel/team2/sandbox/entity/EventEntity.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Data
+@NoArgsConstructor
```
Comment (akarzhaneuski) [inline sandbox-backend/src/main/java/com/exadel/team2/sandbox/entity/EventEntity.java:16]: Only ``` @Data ``` annotation is required here. Another are optional, please check lombok features https://projectlombok.org/features/all and remove unused annotations

File: sandbox-backend/src/main/java/com/exadel/team2/sandbox/entity/EventEntity.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+    @Column(name = "EV_SHORT_DESCRIPTION")
+    private String evShortDescription;
+
+    @Column(name = "EV_FULL_DESCRIPTION")
+    private String evFullDescription;
+
+    @Column(name = "EV_START_DATE")
+    @Temporal(TemporalType.DATE)
+    private Date evStartDate;
```
Comment (akarzhaneuski) [inline sandbox-backend/src/main/java/com/exadel/team2/sandbox/entity/EventEntity.java:47]: LocalDateTime will be better here

File: sandbox-backend/src/main/java/com/exadel/team2/sandbox/entity/EventEntity.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    @Column(name = "EV_START_DATE")
+    @Temporal(TemporalType.DATE)
+    private Date evStartDate;         
+
+    @Column(name = "EV_DURATION")
+    private String evDuration;          //date or String?
+
+    @Column(name = "EV_DEADLINE")
+    @Temporal(TemporalType.DATE)
+    private Date evDeadline;
```
Comment (akarzhaneuski) [inline sandbox-backend/src/main/java/com/exadel/team2/sandbox/entity/EventEntity.java:54]: LocalDateTime

File: sandbox-backend/src/main/java/com/exadel/team2/sandbox/entity/EventTypeEntity.java [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Data
+@NoArgsConstructor
```
Comment (akarzhaneuski) [inline sandbox-backend/src/main/java/com/exadel/team2/sandbox/entity/EventTypeEntity.java:16]: Review annotations here

**Review by akarzhaneuski — CHANGES_REQUESTED** [review]

File: sandbox-backend/src/main/java/com/exadel/team2/sandbox/entity/EventEntity.java (hunk 1 above)
Comment (VadimBlagiy) [reply sandbox-backend/src/main/java/com/exadel/team2/sandbox/entity/EventEntity.java:16]: Done

File: sandbox-backend/src/main/java/com/exadel/team2/sandbox/entity/EventEntity.java (hunk 2 above)
Comment (VadimBlagiy) [reply sandbox-backend/src/main/java/com/exadel/team2/sandbox/entity/EventEntity.java:47]: Done

File: sandbox-backend/src/main/java/com/exadel/team2/sandbox/entity/EventEntity.java (hunk 3 above)
Comment (VadimBlagiy) [reply sandbox-backend/src/main/java/com/exadel/team2/sandbox/entity/EventEntity.java:54]: Done

File: sandbox-backend/src/main/java/com/exadel/team2/sandbox/entity/EventTypeEntity.java (hunk 4 above)
Comment (VadimBlagiy) [reply sandbox-backend/src/main/java/com/exadel/team2/sandbox/entity/EventTypeEntity.java:16]: Done

**Review by ikantsevich — APPROVED** [review]

**Review by akarzhaneuski — APPROVED** [review]

**Pull request merged**